### DISCUSSION TITLE
In-VR editing in Spoke?

### DISCUSSION THREAD

**Discussion by hollyhudson** [mozilla/hubs#3065] [General Questions & Discussion] [answered]
Is there a VR mode for Spoke? When using the editor in firefox, SteamVR stops Steam Home and says "Next up: Firefox", but it doesn't ever display a virtual world in the headset. Is there a hot key to enable VR preview or navigation/interaction with the hand controllers?
(WebXR works on this system; Hubs and a-frame vr websites work fine with the headset)

**Comment by johnshaughnessy** [comment, marked as answer]
@hollyhudson There is not currently a mode for editing Spoke scenes from within VR. I'm surprised that SteamVR is reacting to Spoke because that sounds like [this bug that was fixed a while ago](https://github.com/mozilla/Spoke/issues/665).

A VR editing mode sounds pretty great though, so I would encourage you to [submit this as a feature request in the Spoke repo](https://github.com/mozilla/spoke/issues).
